uefi/src/lib.rs

```diff
@@ -89,8 +89,10 @@
 #![no_std]
 // Enable some additional warnings and lints.
 #![warn(clippy::ptr_as_ptr, missing_docs, unused)]
-#![deny(clippy::all)]
```

Comment:
`clippy:all` is dropped, is that intended?

Reply:
Yikes. Thanks